src/lines.jl

```diff
@@ -1,7 +1,24 @@
+function hasReverseEdge(g, e)
+	return has_edge(g,dst(e),src(e))
+end
+
+function filterUndef(array)
```

Comment:
```suggestion
function filterAssigned(array)
```
or `filterDefined`.

Reason is, that when one uses the `filter` function then predicate specifies what should stay in the array and not what should be filtered out.